.github/workflows/interfaces.yml

```diff
@@ -0,0 +1,30 @@
+name: interfaces
+on:
+  # Trigger at every UTC noon, or manually.
+  schedule:
+    - cron: 0 12 * * *
+  workflow_dispatch:
+jobs:
+  update:
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Run interfaces_update.sh
+      run: ./tools/ci/interfaces_update.sh
+    - name: Create pull request
+      uses: peter-evans/create-pull-request@v3
+      with:
+        # env.webref_idl_version is set by interfaces_update.sh
+        title: "Sync interfaces/ with @webref/idl ${{ env.webref_idl_version }}"
+        commit-message: "Sync interfaces/ with @webref/idl ${{ env.webref_idl_version }}"
+        body: |
+          This PR was automatically created by a bot.
+
+          Before merging, please check that any tests that depend on the updated IDL files still work.
+
+          If additional changes are needed, please manually create another PR based on this one.
+
+          See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.
+        branch: webref/idl
```

Comment:
shouldn't you use a different branch for each pull request? or is it on purpose to avoid creating multiple PRs if they're not getting merged in a timely fashion? Will the branch be deleted on merging?

Reply:
The default behavior is to only ever create one pull request, which seems to be identified by the branch name. I like that  so that pull requests don't pile up. I didn't bother with the `delete-branch` option because it will be deleted when merging the pull request per repo settings, and deleting it in the rare case where the PR was closed due to there being no remaining diff doesn't seem important. The branch would soon be created again anyway.